src/helpers/utils.js

```diff
@@ -0,0 +1,200 @@
+import fs from "fs";
+import { 
+  createDBs, 
+  queryTable, 
+  updateTable 
+} from "./db.js";
+import { sendJobRun } from "./chainlink.js";
+import { MiddlewareENV } from './MiddlewareEnv.js';
+import { logger } from "./logger.js";
+import { FeedsConfig } from "./FeedsConfig.js";
+
+// Load environment variables
+let envvars = {};
+try{
+  envvars = new MiddlewareENV();
+} catch (err) {
+  if (process.env.NODE_ENV !== "test" && process.env.SERVICE !== "monitor") {
+    logger.error("ERROR LOADING ENV VARS: " + err)
+    process.exit(1);
+  }
+}
+
+/**
+ * Function to read a json file
+ * @param {string} filename  file name or path to read
+ * @returns {object} the JSON data in the file
+ */
+export const readJSONFile = (filename) => {
+  try{
+    let rawdata = fs.readFileSync(filename);
```

Comment:
change to rawData

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/29244ced3c6d34785c4643507480179d98d7d8c6